src/components/notifications/AppSelector/index.tsx

```diff
@@ -136,6 +137,7 @@ const AppSelector: React.FC = () => {
                               app.metadata.icons?.length ? app.metadata.icons[0] : '/fallback.svg'
```

Comment:
Couldn't this `<img>` be inside a component to avoid so many duplicate implementations? And prevent future fallback image issues?

Reply:
Yes - definitely a much needed refactor. However, not in scope for this PR.

Reply:
Sounds good. Oops, forgot to approve with my comment